css/fonts.css

```diff
@@ -1,15 +1,24 @@
 @font-face {
-	font-family: 'Gotham Pro';
-	src: url('GothamPro/GothamPro-Medium.woff') format("woff"),
-	url('GothamPro/GothamPro-Medium.woff2') format("woff2");
-	font-style: 500;
-	font-weight: normal;
+  font-family: 'Gotham Pro';
+  src: url('GothamPro/GothamPro-Medium.woff') format("woff"),
+    url('GothamPro/GothamPro-Medium.woff2') format("woff2");
+  font-style: normal;
+  font-weight: normal;
 }
 
 @font-face {
-	font-family: 'Gotham Pro';
-	src: url('GothamPro/GothamPro-Bold.woff') format("woff"), 
-	url('GothamPro-Bold.woff2') format("woff2");
-	font-style: normal;
-	font-weight: bold;
+  font-family: 'Gotham Pro';
+  src: url('GothamPro/GothamPro-Bold.woff') format("woff"),
+    url('GothamPro-Bold.woff2') format("woff2");
+  font-style: normal;
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: 'Gotham Pro';
+  src: 
+  url('GothamPro/GothamPro-Light.woff') format("woff"),
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/axi/pullRequest?prid=6140750)